docs/artifactory/refreshtoken/help.go

```diff
@@ -0,0 +1,12 @@
+package refreshtoken
+
+const Description = "Refresh access token."
+
+var Usage = []string{"jfrog rt refresh-token"}
+
+const Arguments string = `	refresh token
+		The refresh token of the access token that needs to be refreshed.
```

Comment:
The refresh token of the access token that needs to be refreshed. --> The refresh token used for refreshing the access token.